migrate.py

```diff
@@ -1659,21 +1659,66 @@ def integration_tests_add_to_deps(collection, dep_collection):
     integration_tests_deps.add(dep_collection)
 
 
-def poor_mans_integration_tests_discovery(checkout_dir, plugin_type, plugin_name):
-    # FIXME this might be actually enough for modules integration tests, at least for the most part
-    if plugin_type != 'modules':
-        return []
+def integration_tests_discovery(checkout_dir, plugin_type, plugin_name):
```

Comment:
Let's rename this to
```suggestion
def discover_integration_tests(checkout_dir, plugin_type, plugin_name):
```